cli/main.rs

```diff
@@ -513,19 +518,11 @@ async fn run_command(flags: Flags, script: String) -> Result<(), ErrBox> {
   let mut worker =
     create_main_worker(global_state.clone(), main_module.clone())?;
   debug!("main_module {}", main_module);
+  write_lockfile(global_state)?;
```

Comment:
Modules are not yet loaded at this point; this call should be done after `worker.execute_module(&main_module).await?;`

Reply:
Ah I see. Unfortunately this brings up the old problem, with `worker.execute_module` running the long-lived code and blocking the lockfile from being written.

Reply:
That would only happen in case of code with busy-loop inside. If it say server then it will execute the code, call `load` event and start awaiting promises. We want to write down lock file just before `load` event.

Reply:
Ahh I see now, ok! I've made the changes. Thank you for the very helpful feedback!